3p/bodymovinanimation.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {loadScript} from './3p';
+import {parseJson} from '../src/json';
+
+/**
+ * Produces the AirBnB Bodymovin Player SDK object for the passed in callback.
+ * @param {!Window} global
+ * @param {function(!Object)} cb
+ */
+
+function getBodymovinAnimationSdk(global, cb) {
```

Comment:
nit: can we move private methods below public methods?

Reply:
Done!

Reply:
Had to undo that - if you reverse the order the `gulp check-types` run fails  because `cb` is undefined.